src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -45,7 +45,7 @@
      * Initializes a ModelManager with the given addressBook, userPrefs and commandHistory.
      */
     public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs,
-            ReadOnlyCommandHistory commandHistory) {
+                        ReadOnlyCommandHistory commandHistory) {
```

Comment:
Is the spacing here supposed to be 8 spaces instead of 16?

Reply:
This was automatically done by the code formatter, and checkstyle seems ok with it. It seems to follow the "conventional indentation" guidelines set out by oracle: https://www.oracle.com/java/technologies/javase/codeconventions-indentation.html

However, the textbook (https://se-education.org/guides/conventions/java/intermediate.html#layout) says line breaks should only double the indentation. Let me know what you want to do about this.